### ISSUE TITLE
Is this usable?

### ISSUE THREAD

**Issue by markstuttword** [RDFLib/rdflib-rdfstar#13]
Hello,
since this looks like to be only existing attempt at RDF* in Python, I was wondering if it's already usable.
Otherwise I'd really like to know if there are other working Python libraries for RDF* to use until RDFLib implements it.
Thank you.

**Comment by gjhiggins** [comment]
Me again, sorry :smile: No, not in a usable state. it's an in-progress [student research project](https://cecs.anu.edu.au/research/student-research-projects/rdf-star-reference-implementation)

**Comment by alooba1** [comment]
yeah, I am still fixing stuffs, it should be usable in the second half of this year

**Comment by valecarriero** [comment]
Hi there, I would love to use this library! Is it usable now? :)

**Comment by XuguangSong98** [comment]
@valecarriero this work is move to this repo https://github.com/XuguangSong98/rdflib
Here is the pull request
https://github.com/RDFLib/rdflib/pull/2115.
yeah the pull request is still pending, but the Turtle* Ntriples*, Trig* parsers and Turtle*, Trig* serializer is complete, I will be working on fixing the CI failures and codes styles to make it pass the pull request. SPARQL* can be found on the sparql-star branch but only works for some basic select queries insert/delete and I am fixing the where clause. So, it should be usable in this month

**Comment by sdasda7777** [comment]
Hi, this project seems very promising and I'd love to use it. What seems to be the current status? Are there currently any major problems to be aware of?
